### PR TITLE
Adding the commonUtil.is_video_recording for checking video recorders

### PR DESCRIPTION
**Pull request by askeing**

### CODE REVIEW

**Comment by askeing** [comment]
@ShakoHo @MikeLien r?

File: lib/common/commonUtil.py [hunk 1]
```diff
@@ -42,3 +43,24 @@ def atoi(text):
...
+        """
+        Checking the Video Recording settings.
+        @return: True if there is one Recorder, False if there is no Recorder. Raise exception when there are multiple Recorders.
+        """
+        recording_flags = {Environment.PROFILER_FLAG_AVCONV: False}
+        for k in recording_flags.keys():
+            v = settings.get(k, {})
+            if v:
+                is_enabled = v.get('enable', False)
+                recording_flags[k] = is_enabled
```
Comment (ypwalter) [inline lib/common/commonUtil.py]: why don't we use
`recording_flags[k] = v.get('enable', False)`

Comment (askeing) [inline lib/common/commonUtil.py]: I think it will be more readable than one line code :P

**Comment by askeing** [comment]
@ShakoHo r? again XD

**Comment by ShakoHo** [comment]
lgtm r+

**Pull request merged**